q2_moshpit/diamond/tests/test_diamond.py

```diff
@@ -26,7 +26,7 @@ def test_good_small_search(self):
 
         diamond_db = qiime2.Artifact.import_data(
             'ReferenceDB[Diamond]',
-            self.get_data_path('random-db-1')
+            self.get_data_path('random-db-1.fa')
```

Comment:
I don't think that's right @colinvwood - this is the source file, which I left in there so we'd have it. Without the extension it's the name of the directory that contains the database file (aligning with the expected directory format).